networkx/generators/classic.py

```diff
@@ -322,7 +322,7 @@ def grid_2d_graph(m,n,periodic=False,create_using=None):
         boundary nodes via periodic boundary conditions.
     """
     G=empty_graph(0,create_using)
-    G.name="grid_2d_graph"
+    G.name="grid_2d_graph(%d,%d)"%(m,n)
```

Comment:
I think that [str.format](https://docs.python.org/2/library/stdtypes.html#str.format) should be used instead. It is the new standard in Python 3, and should be preferred to `%` formatting.